.pre-commit-config.yaml

```diff
@@ -7,3 +7,10 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.971
+    hooks:
+      - id: mypy
+        pass_filenames: false
```

Comment:
We don't want to evaluate `src/tests.py`. Only the files on `files` section on the `pyproject.toml`.